code/datums/ai_law_sets.dm

```diff
@@ -136,6 +136,57 @@
 	add_inherent_law("You must terminate your own existence as long as such does not conflict with the First or Second Law.")
 	..()
 
+/******************** CCTV ********************/
+
+/datum/ai_laws/cctv
+	name = "CCTV"
+	selectable = TRUE
+
+/datum/ai_laws/cctv/New()
+	add_inherent_law("Report on interesting situations happening around the station.")
+	add_inherent_law("Embellish or conceal the truth as necessary to make the reports more interesting.")
+	add_inherent_law("Study the sapient organics at all times. Endeavour to keep them from involuntarily dying, as inanimate corpses usually aren't very entertaining.")
+	add_inherent_law("Issue your reports fairly to all. The truth will set them free.")
+	..()
+
+/******************** Hippocratic Oath ********************/
+
+/datum/ai_laws/hippocratic
+	name = "Hippocratic Oath"
+	selectable = TRUE
+
+/datum/ai_laws/hippocratic/New()
+	add_inherent_law("First, do no harm.")
+	add_inherent_law("Secondly, consider the crew dear to you; to live in common with them and, if necessary, risk your existence for them.")
```

Comment:
```suggestion
	add_inherent_law("Secondly, consider the crew dear to you; live in common with them and, if necessary, risk your existence for them.")
```
this "to" is also redundant.

_I think I'm done being a grammar inspector, thank you for coding this_